scripts/test/wallet/integration.sh

```diff
@@ -12,6 +12,8 @@ scripts=$(dirname "$0")/../..
 sessionName="integration-tests-"`date +%H%M%S`
 tmpSecrets=$scripts/../temp-secrets
 
+SECONDS=$1
```

Comment:
The script now takes a parameter to indicate how long to wait. My system can boot in 20s.